### PR TITLE
[Don't merge till travis passes]Improve help docs and fix related test cases

### PR DESCRIPTION
**Pull request by gimantha**
## Purpose
$subject

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Required Balo version update
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-service.help [hunk 1]
```diff
@@ -1,26 +1,42 @@
...
+       <openapi-contract>
+           The contract file location of the OpenAPI contract, which
+           is used to create the Ballerina service.
+
+       -o, --output <dir-path>
+           Output directory in which service should be generated. If this options
+           is not provided, the service is generated in the current directory.
+
+       -c, --skip-bind
+           Copy the OpenAPI contract file to the projects resources directory
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-service.help]:
```suggestion
           Copy the OpenAPI contract file to the project's resources directory
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi.help [hunk 2]
```diff
@@ -7,24 +7,25 @@ NAME
...
+                                    <openapi-contract> [-o <dir-path> | --output <dir-path>]
     
-       ballerina openapi gen-contract [<module>:]<service-name>
-                                      [-i <ballerina-file>]
-                                      [-o <contract-file>]
-                                      [-s|--skip-bind]
+       ballerina openapi gen-contract [<module-name>:]<service-name>
+                                      [-i <ballerina-file> | --ballerina-file <ballerina-file>]
+                                      [-o <openapi-contract> | --contract-location <openapi-contract>]
+                                      [-s| --skip-bind]
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi.help]:
```suggestion
                                      [-s | --skip-bind]
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-remove.help [hunk 3]
```diff
@@ -1,5 +1,5 @@
 NAME
-    Ballerina remove - Remove a given distribution version
+    Ballerina-remove - Remove a given distribution version
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-remove.help]:
```suggestion
    ballerina-remove - Remove a given distribution version
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help [hunk 4]
```diff
@@ -56,25 +56,25 @@ OPTIONS
 
 
 DEFAULT BEHAVIOR
-       Runs the 'main' function in the given program or module, if present.
+       Run command run the 'main' function in the given program or module, if present.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help]:
```suggestion
       Run command runs the 'main' function in the given program or module, if present.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-search.help [hunk 5]
```diff
@@ -6,7 +6,7 @@ SYNOPSIS
 
 
 DESCRIPTION
-       Searches Ballerina Central for the given keyword appearing in
+       Search command searches the Ballerina Central for the given keyword appearing in
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-search.help]:
```suggestion
       Search command searches Ballerina Central for the given keyword appearing in
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-uninstall.help [hunk 6]
```diff
@@ -1,13 +1,13 @@
 NAME
-       Ballerina uninstall - Uninstall modules from the home repository
+       Ballerina-uninstall - Uninstall modules from the home repository
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-uninstall.help]:
```suggestion
       ballerina-uninstall - Uninstall modules from the home repository
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-update.help [hunk 7]
```diff
@@ -1,15 +1,15 @@
 NAME
-    Ballerina dist update - Update to latest distribution version
+    Ballerina-dist-update - Update to latest distribution version
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-update.help]:
```suggestion
    ballerina-dist-update - Update to latest distribution version
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-use.help [hunk 8]
```diff
@@ -1,15 +1,15 @@
 NAME
-    Ballerina dist use - Use a given distribution version
+    Ballerina-dist-use - Use a given distribution version
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-use.help]:
```suggestion
    ballerina-dist-use - Use a given distribution version
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-use.help [hunk 9]
```diff
@@ -1,15 +1,15 @@
...
 
 SYNOPSIS
     ballerina dist use <distribution-version>
 
 DESCRIPTION
-     Pull and use a given Ballerina version from project repository.
+     Use pulls and use a given Ballerina version from project repository.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-use.help]:
```suggestion
     Use pulls and uses a given Ballerina version from project repository.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 10]
```diff
@@ -1,43 +1,44 @@
...
-           [Optional] The module name to use when generating the client.
 
-       service-name
-           The service name to use when generating the client.
-           This option will require either a valid Ballerina file
-           or a valid Ballerina module.
+OPTIONS
+       [<module-name>:]<service-name>
+           The service name to use when generating the contract.
+           This option will require a valid Ballerina file with an optional module name.
```
[the remark this answers is not in the document]
Comment (gimantha) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]: fixed

**Pull request merged**